client/src/styles/app.scss

```diff
@@ -42,9 +38,9 @@ $sizes: (
 }
 
 body {
-  background-color: $burgundy;
-  //background-image: url('@/assets/images/background.svg');
-  //background-size: 100vw 100vh;
+  background-color: $white;
```

Comment:
Set this back to burgundy, if the image doesn't load we don't want the user to blinded.